Sources/securekeystore/SecureKeystoreImpl.swift

```diff
@@ -176,24 +176,23 @@ public class SecureKeystoreImpl:SecureKeystoreProtocol {
     
     public func encryptData(alias:String, data:String, onSuccess: @escaping (String) -> Void, onFailure: @escaping(_ code: String,_ message: String)->Void)
     {
-        aesCryptoManager.encrypt(keyTag: alias, data: data){
-            cipher in if(cipher != nil){
-                onSuccess(cipher!)
+        aesCryptoManager.encrypt(keyTag: alias, data: data){ text in
```

Comment:
Can we use variables which convey better use/meaning of the variable? like `result` , `errorMessage` instead of `text`

Reply:
changed it to result, as it may contain the data or error